### PR TITLE
[CLI Config Parity] Migrate Ignore Interrupts config and flag. 

### PR DESCRIPTION
**Pull request by ashmeenkaur**
### Description
This PR migrate Ignore Interrupts config and flag. Also added tests for file system config.
I will make kernel list cache and Enable Parallel Dirops migration changes in a follow up PR.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - Added
3. Integration tests - Via KOKORO

### CODE REVIEW

File: cmd/config_validation_test.go [hunk 1]
```diff
@@ -402,3 +407,86 @@ func TestValidateConfigFile_GCSConnectionConfigSuccessful(t *testing.T) {
...
+		name           string
+		configFile     string
+		expectedConfig *cfg.Config
+	}{
+		{
+			name:       "Empty config file [default values].",
+			configFile: "testdata/empty_file.yaml",
+			expectedConfig: &cfg.Config{
+				FileSystem: cfg.FileSystemConfig{
+					DirMode:                cfg.Octal(defaultDirMode),
```
Comment (kislaykishore) [inline cmd/config_validation_test.go:432]: Ditto for other cases
```suggestion
					DirMode:               0755,
```

**Review by kislaykishore — CHANGES_REQUESTED** [review]

File: cmd/config_validation_test.go [hunk 2]
```diff
@@ -237,6 +237,10 @@ func TestValidateConfigFile_InvalidConfigThrowsError(t *testing.T) {
 			name:       "Invalid value of anonymous access.",
 			configFile: "testdata/gcs_auth/invalid_anonymous_access.yaml",
 		},
+		{
+			name:       "Invalid value of ignore interrupts.",
```
[the remark this answers is not in the document]
Comment (kislaykishore) [reply cmd/config_validation_test.go]: Just checking if this was missed.

[the remark this answers is not in the document]
Comment (ashmeenkaur) [reply cmd/config_validation_test.go]: Done, sorry forgot to resolve.

File: cmd/root_test.go [hunk 3]
```diff
@@ -560,3 +560,122 @@ func TestArgsParsing_GCSConnectionFlagsThrowsError(t *testing.T) {
 		})
 	}
 }
+
+func TestArgsParsing_FileSystemFlags(t *testing.T) {
+	hd, err := os.UserHomeDir()
+	require.NoError(t, err)
+	require.NoError(t, err)
```
Comment (kislaykishore) [inline cmd/root_test.go:567]:
```suggestion
```

**Review by kislaykishore — CHANGES_REQUESTED** [review]

**Review by kislaykishore — APPROVED** [review]

**Pull request merged**